docs/adding_new_platform_guide.md

```diff
@@ -0,0 +1,93 @@
+# Adding a new platform to Jetstream
+
+Steps for addinga new platform include:
+
+1. Changes are made in Experimenter
+2. Adding support in mozanalysis
+3. New mozanalysis release
+4. Update mozanalysis in jetstream
+5. Add support for platform in jetstream
+6. Tests
+
+## Changes are made in Experimenter
+https://github.com/mozilla/experimenter
+
+- Follow the [run an experiment guide](https://experimenter.info/#run-an-experiment) for creating a new experiment
+- https://experimenter.info/jetstream/configuration#experiment-section
+
+---
+## Adding support in mozanalysis
+https://github.com/mozilla/mozanalysis/
+- Add new metric(s) in `mozanalysis` (if needed). For more detailed guide follow [how do I add a metric to my experiment](https://experimenter.info/jetstream/metrics#how-do-i-add-a-metric-to-my-experiment), and [defining metric](https://experimenter.info/jetstream/configuration#metrics-section)
+- Add new segment(s) in `mozanalysis` (if needed). For more detailed guide follow [defining segment](https://experimenter.info/jetstream/configuration#defining-segments)
```

Comment:
- Default `metric`s and `datasource`s need to be added to [mozanalysis](https://github.com/mozilla/mozanalysis/blob/main/src/mozanalysis/metrics). These default metric and data source definitions will be available for [custom experiment configurations](https://experimenter.info/jetstream/configuration/)
- Optionally, `segment` definitions can also be added in [mozanalysis](https://github.com/mozilla/mozanalysis/tree/main/src/mozanalysis/segments)